GLAA.Web/Views/Shared/DisplayTemplates/EligibilityViewModel.cshtml

```diff
@@ -2,28 +2,45 @@
 
 <table class="check-your-answers">
     <thead>
-        <tr>
-            <th colspan="3">Eligibility</th>
-        </tr>
+    <tr>
+        <th colspan="3">Account set up</th>
+    </tr>
     </thead>
     <tbody>
     <tr>
-        <td>@Html.LabelFor(m => m.SuppliesWorkers.SuppliesWorkers)</td>
-        <td class="@Model.SuppliesWorkers?.SuppliesWorkers">@Html.DisplayFor(m => m.SuppliesWorkers)</td>
-        <td class="change-answer"><a href="@Url.Action("Part1", "Eligibility")">Change<span class="visually-hidden"> @Html.LabelFor(m => m.SuppliesWorkers.SuppliesWorkers)</span></a></td>
+        <td>Principal Authority's email address</td>
+        <td class="@Model.EmailAddress?.EmailAddress">@Html.DisplayFor(m => m.EmailAddress.EmailAddress)</td>
+        <td class="change-answer">
+            <a asp-controller="Eligibility" asp-action="Eligibility" asp-route-id="1">Change<span class="visually-hidden"> Principal Authority's email address</span></a>
```

Comment:
.net core me up! I will try to use these going forward but I didn't have the heart (or time) to rework all the html helpers...one day

Reply:
Thing is it's less compact than using good old `Url.Action()`! The dynamic route parameter attribute is really nice though.